internal/policies/manager.go

```diff
@@ -291,19 +291,23 @@ func NewManager(bus *dbus.Conn, hostname string, opts ...Option) (m *Manager, er
 
 // ApplyPolicies generates a computer or user policy based on a list of entries
 // retrieved from a directory service.
-func (m *Manager) ApplyPolicies(ctx context.Context, objectName string, isComputer bool, pols *Policies) (err error) {
+func (m *Manager) ApplyPolicies(ctx context.Context, objectName string, isComputer, purge bool, pols *Policies) (err error) {
```

Comment:
I’m unsure about this additional parameter, because I can see 2 advantages of using pols being empty:
- then, this directly translates to the managers themselves, when we pass an empty slice for purging.
- we can map that when we switch from "Pro to non Pro machines" to purge the additional policies (still applying dconf/gdm)

Also, that removes any notion of "purge" in `internal/policies`, it’s just updating with no policies.
What do you think?